pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>CMAT</groupId>
     <artifactId>MeasureAuthoringTool</artifactId>
-    <version>5.8</version>
+    <version>6.0</version>
```

Comment:
This needs to be switched back to 5.8, waiting to see if other parts of this file need updated/changed too.